manuals/2.0/ja/setup.md

```diff
@@ -2,7 +2,7 @@
 layout: docs2-ja
 title: Setting up your virtual host
 permalink: /manuals/2.0/ja/setup/
-previous_page: Models
+previous_page: Command line / console
```

Comment:
Yes.

See https://raw.githubusercontent.com/auraphp/auraphp.github.com/master/manuals/2.0/en/setup.md and the next line `previous_page_url`.

`Models` is simply wrong.